mex/common/models/__init__.py

```diff
@@ -44,6 +45,8 @@
 - `_VariadicValues` re-defines all fields from `_OptionalValues` and `_RequiredValues`
   as list fields with an arity of 0-n
 
+- `_BaseRuleSet` bundles the additive, subtractive and preventive rules for one type
```

Comment:
This looks like a private variable. How about something like `{EntityType}BaseRuleSet`, maybe along with an example?

Reply:
they are private classes and they are named exactly `_BaseRuleSet` for all entity types, because they shouldn't "leave" their respective modules anyway.

the public ones are mentioned below --> https://github.com/robert-koch-institut/mex-common/pull/258/files#diff-fadf4248d60efe4368000c6681f84bd2dfe0b75aabc68c6b371b01438227e29eR59-R60

Reply:
Alright, thanks for clearing that up. My rephrased question is then: why do we document private classes in the public docs?

Reply:
to help the reader understand how the class composition for the public classes works.

this is the same across all 12 entity-types, and i'd rather not copy&paste this 12 times